loadgen/test_settings.h

```diff
@@ -233,6 +233,16 @@ struct TestSettings {
   /// log in performance mode
   double accuracy_log_probability = 0.0;
   /**@}*/
+  
+  // ==================================
+  /// \name Performance Sample modifiers
+  /// \brief Allows issuing only unique samples in Performance mode of any scenario
+  bool issue_unique = false;
+  /// \brief The Same sample is chosen repeatedley for Inference
+  bool issue_same = false;
```

Comment:
issue_same -> performance_issue_same

Reply:
I think, in addition to the flag, it will be useful to have an offset called performance_issue_same_index to control which sample is repeated.